src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java

```diff
@@ -52,33 +52,101 @@ void redirectToMyMDTHINKOnUnsupportedCounty() {
     }
 
     @Test
-    void selectHelpNeededAndChooseProgramFlow() {
+    void selectNotSureHelpNeededAndChooseProgramFlow() {
         // select help needed
         testPage.navigateToFlowScreen("mdBenefitsFlow/selectHelpNeeded");
         testPage.clickContinue();
         assertThat(testPage.getTitle()).isEqualTo("Select help");
-        assertThat(testPage.hasErrorText(message("error.missing-general")));
+        assert(testPage.hasErrorText(message("error.missing-general")));
+
+        testPage.clickElementById("helpNeeded-CHILDREN");
```

Comment:
I think you can take this line out -- or comment that it does nothing, as the checking `none__checkbox` will cause it to get unchecked.   And maybe that's part of the test as well?

Reply:
- I added a comment since I believe that this is actually an important part of the test since none__checkbox unsets the other checkboxes and behaves differently in the programs pages.